### PR TITLE
add mType to sim/Morphology and ontology shapes for eType/mType (#109)

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/morphology/v0.1.3/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1",
```
Comment (MFSY) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json]: I suggest you create a new version of {{base}}/schemas/neurosciencegraph/commons/modelinstance and import {{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.2 their or in both places.

Comment (genric) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json]: Don't understand why do we need new modelinstance?
And why did you merge master into simulation_schemas? I thought we agreed to rebase it on master after we are happy with it and merge changes to master.

Comment (MFSY) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json]: Here is the issue:

* {{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1 imports {{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.1.

* in neurosciencegraph/simulation/morphology/v0.1.3.json you want to import {{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.2.

* So two different versions of {{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm are imported in the same scope. That's an issue.

About merging master into simulation_schemas. You're right, I completely forgot that the changes come from another branch (sort of merge reflex).

Comment (genric) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json]: ok, fix is in https://github.com/INCF/neuroshapes/pull/118

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  "shapes": [
+    {
+      "@id": "this:MorphologyShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "targetClass": "nsg:Morphology",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
```
Comment (MFSY) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json]: Update to {{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.2

Comment (genric) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json]: fixed: https://github.com/INCF/neuroshapes/pull/119

**Pull request merged**